generator/src/main/java/com/scottlogic/deg/generator/validators/GenerationConfigValidator.java

```diff
@@ -1,15 +1,19 @@
 package com.scottlogic.deg.generator.validators;
 
 import com.scottlogic.deg.generator.generation.GenerationConfig;
+import com.scottlogic.deg.generator.outputs.targets.FileOutputTarget;
+import com.scottlogic.deg.generator.outputs.targets.OutputTarget;
 
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.ArrayList;
 
 /**
  * Class used to determine whether the command line options are valid for generation
  * */
 public class GenerationConfigValidator {
 
-    public ValidationResult validateCommandLine(GenerationConfig config) {
+    public ValidationResult validateCommandLine(GenerationConfig config, OutputTarget fileOutputTarget) {
```

Comment:
See previous comment about whether this should be an argument or a dependency.

Reply:
changed to dependency